src/main/features/response/routes/statement-of-means/bankAccounts.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from 'express'
+
+import { Paths, StatementOfMeansPaths } from 'response/paths'
+import { Form } from 'forms/form'
+import { FormValidator } from 'app/forms/validation/formValidator'
+import { ErrorHandling } from 'common/errorHandling'
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { RoutablePath } from 'common/router/routablePath'
+import { BankAccounts } from 'response/form/models/statement-of-means/bankAccounts'
+
+const page: RoutablePath = StatementOfMeansPaths.bankAccountsPage
+
+function renderView (form: Form<BankAccounts>, res: express.Response): void {
+  res.render(page.associatedView, { form: form })
+}
+
+function actionHandler (req: express.Request, res: express.Response, next: express.NextFunction): void {
+  if (req.body.action) {
+    const form: Form<BankAccounts> = req.body
+    if (req.body.action.addRow) {
+      form.model.appendRow()
+    }
+    return renderView(form, res)
+  }
+  next()
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(page.uri, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
Please add `FeatureToggleGuard.featureEnabledGuard('statementOfMeans')` here and for POST handler.

Reply:
ok